code/modules/reagents/chemistry/reagents/drink_base.dm

```diff
@@ -23,4 +23,4 @@
 	if(adj_temp_cool)
 		if(M.bodytemperature > 310)//310 is the normal bodytemp. 310.055
 			M.bodytemperature = max(310, M.bodytemperature - (adj_temp_cool * TEMPERATURE_DAMAGE_COEFFICIENT))
-	..()
\ No newline at end of file
+	..()
```

Comment:
The fact that you added a newline here makes me assume you haven't disabled the stupid automatic behaviour in Atom over it; You should really go to `Settings > Packages > Whitespace` and uncheck `Ensure Single Trailing Newline`.